### ISSUE TITLE
Make native tests "image wait time" configurable

### ISSUE THREAD

**Issue by gwenneg** [quarkusio/quarkus#3413]
**Description**
The current [NativeImageLauncher](https://github.com/quarkusio/quarkus/blob/master/test-framework/common/src/main/java/io/quarkus/test/common/NativeImageLauncher.java#L24) code waits 60 seconds for the native image to start before failing the tests. This should be configurable to allow slow computers to run native tests.

**Implementation ideas**
The native image wait time configuration would look just like the test http port configuration. However, I'm not sure where this new `quarkus.test.native-image-wait-time` property would go. I didn't find any native-dedicated `@ConfigRoot` class in Quarkus. There's only an existing `quarkus.native.enable-all-charsets` property which does not seem to belong to any `@ConfigRoot` class. Should the image wait time property be used the same way without being part of a config class?

**Comment by gwenneg** [comment]
I tried to run native tests from a quarkus-quickstarts modified project using my PTO not-that-slow MacBook and it took `7,403,556.61 ms` for the native image to start.

**Comment by gwenneg** [comment]
I ended up running native tests from another quickstarts project without Docker and the native image started much faster (below 60 seconds), so I no longer need to make the image wait time configurable.

I'm not sure what caused the original issue (with the huge wait time) but I reproduced it from several quickstarts projects with much lower wait times (but still over 60 seconds). Since I've already made the code changes and it could be useful for someone else, I'll submit a PR.
